internal/core/metadata/application/notify.go

```diff
@@ -46,6 +50,8 @@ func addDeviceCallback(ctx context.Context, dic *di.Container, device dtos.Devic
 	if response.StatusCode != http.StatusOK {
 		lc.Errorf("fail to invoke device service callback for adding device %s, err: %s", device.Name, response.Message)
 	}
+
+	sendNotification(ctx, dic, device.Name, http.MethodPost)
```

Comment:
Use `Device creation` instead of `http.MethodPost`

Reply:
Modified.